tests/io/mscw_ctm/test_aeroval_configs.py

```diff
@@ -0,0 +1,13 @@
+from pyaerocom.aeroval.config.pmratios.base_config import get_CFG
+
+
+def test_ratpmconfig():
+    """short test if the example configuration for pm ratios is still in the code"""
+
+    reportyear = year = 2019
```

Comment:
Would be nice to include a test which runs the config with coarse data and checks that it works as expected.

Reply:
made this a separate issue #1379